lib/chef/knife/ssh.rb

```diff
@@ -332,7 +332,9 @@ def ssh_command(command, subsession = nil)
         command.force_encoding("binary") if command.respond_to?(:force_encoding)
         subsession.open_channel do |chan|
           if config[:on_error] && exit_status != 0
-            chan.close()
```

Comment:
and this feels like we're workaround around a bug in net-ssh / net-ssh-multi somewhere that should get (or already is?) fixed properly